app/controllers/concerns/importable.rb

```diff
@@ -0,0 +1,28 @@
+module Importable
```

Comment:
Should we add a module description here to explain how this module should be used/included in controllers.
For example, as a newbie to this codebase, I didn't expect `import_csv` to be a route on the controller. Instead, I'd assumed it was a helper of some kind.

For example:

```
# Importable adds an `import_csv` action that that responds to routes like this:
#         resources :storage_locations do
#           collection do
#             post :import_csv
#           end
#         end
#
# If the model doing the import doesn't match the controller's name, you should update the 
# add a `resource_model` method and to override the default class, e.g.
#    class Admin::LocationsController
#      def resource model
#        Location
#      end
#    end

Reply:
Agreed, will add this!